src/application/service_tester.go

```diff
@@ -28,7 +28,7 @@ func NewServiceTester(serviceContext types.ServiceContext, logger *util.Logger,
 	serviceLocation := serviceContext.Location
 	serviceDir := serviceContext.Dir
 	appDir := serviceContext.AppContext.Location
-	dockerComposeProjectName := fmt.Sprintf("%stests", composebuilder.GetDockerComposeProjectName(appDir))
+	dockerComposeProjectName := serviceContext.AppContext.DockerComposeProjectName
```

Comment:
@hugobho this change and updating `AppContext` doesn't seem related to this PR. I don't really like its addition anyway, so please create a new PR where can discuss it in isolation.